types/types.go

```diff
@@ -27,11 +27,23 @@ type NetConf struct {
 	types.NetConf
 	ConfDir    string                   `json:"confDir"`
 	CNIDir     string                   `json:"cniDir"`
-	Delegates  []map[string]interface{} `json:"delegates"`
+	_delegates []map[string]interface{} `json:"delegates"`
```

Comment:
The name '_delegates' and 'Delegates' are bit confusing. How about to change '_delegates' to 'delegatesInJson' or some name indicating 'json'?